### ISSUE TITLE
After member-only room resume from hibernation, room members lose ability to query users (members, admin, owner)

### ISSUE THREAD

**Issue by npsopheak** [esl/MongooseIM#3612]
MongooseIM version: 5.0
Installed from: Docker
Erlang/OTP version: n/a

Hello there,

I encounter an issue that I configure the room with members-only, allowing query members for all affiliations (member, admin, owner).
I have already added members to the list and as owner, I can query the list normally.

Reproduce steps:

Let's say: MUC Group owner: Bob, MUC group member: Alice

1. As an owner (e.g Bob), create a room with room configuration of persistent, members-only, allowing query members to all affiliations (member, admin, owner).
2. As an owner, invites the intended member (e.g. Alice) to the room
3. As a member, Alice, received an invitation, accept the invitation and announce their presence in the group
4. As a member, Alice, could query all member's affiliations, seeing Bob as admin, send a message and receive a message
5. Stop using the group for a long while (e.g. in my case I did not measure the time, usually I left work at 7 pm and start work again at 9 am).
6. As a member, Alice, opened the app and go to the group chat room.
7. As a member, Alice, could send presence, send a message and receive messages.
8. BUT now Alice, could not retrieve all members having issues as below:

```xml

<iq from='demo-room@muc.example.com' to='alice@example.com/123' id='WYRMCVWRB' type='error'>
  <query xmlns='http://jabber.org/protocol/muc#admin'>
    <item affiliation='member'/>
  </query>
  <error code='403' type='auth'>
    <forbidden xmlns='urn:ietf:params:xml:ns:xmpp-stanzas'/>
    <text xmlns='urn:ietf:params:xml:ns:xmpp-stanzas'>Administrator privileges required</text>
  </error>
</iq>

```

I could make Alice query members again by having Bob as owner, to re-send group configuration. I suspect somehow some options of the room configuration got revoked during resuming from hibration. 

P.S. I try removing and adding Alice to the room again but it does not help, only re-send the config.

Could you check what could be the issue?

**Comment by gustawlippa** [comment]
Hi! Thank you for the very detailed description. I don't see right now from where the problem could arise from.

Could you please:
- provide the contents of your config file (`mongooseim.tom`)? Especially the `modules.mod_muc` section.
- just to be sure, could you please show the stanza in which Bob is setting the room config?

**Comment by chrzaszcz** [comment]
This seems to duplicate the closed issue #3705.
